templates/common/_base/units/machine-config-daemon-host.service

```diff
@@ -1,5 +1,5 @@
 name: "machine-config-daemon-host.service"
-enabled: true
+enabled: false
```

Comment:
On purpose or did this sneak in? :slightly_smiling_face:

Reply:
It's from PR https://github.com/openshift/machine-config-operator/pull/1366 , picked it from PR in  advance so that this doesn't cause problem while testing.